<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userList</class>
 <widget class="QDialog" name="userList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="adminRights">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin rights</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="userBlock">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>User blocked</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="validation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Validation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
